packages/ckeditor5-dev-env/lib/release-tools/utils/getpackagestorelease.js

```diff
@@ -37,11 +45,14 @@ module.exports = function getPackagesToRelease( pathsToPackages ) {
 
 		return Promise.resolve();
 	}
-
-	return executeOnPackages( pathsToPackages, filterPackagesToRelease )
-		.then( () => {
-			process.chdir( cwd );
-
-			return Promise.resolve( packagesToRelease );
-		} );
 };
+
+/**
+ * @typedef {Object} ReleaseDetails
```

Comment:
TODO: Add properties: `repositoryOwner`, `repositoryName`, `npmVersion`, `npmRelease`, `githubVersion`, `githubRelease`. 

Question to myself: `(npm|github)Release` should be `is(Npm|Github)Release`?

Reply:
Done. I went with `shouldReleaseOn(Npm|Github)`.